### ISSUE TITLE
Cannot run latest docker image

### ISSUE THREAD

**Issue by hemker** [misery/DockerAusweisApp2#2]
I am unable to run the latest docker image using the provided instructions.
The error I receive is:
```
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, minimal, minimalegl, offscreen, vnc.
```
Enabling the qt debug output, no additional information are printed.
Running the container on an Ubuntu 20.04.

**Comment by misery** [comment]
Thanks, Qt wasn't able to create the xcb plugin. I fixed that in 67fcc90050663d68fd88498417011c4a00fef3e8.

**Comment by misery** [comment]
Seems this was the reason: https://bugreports.qt.io/browse/QTBUG-88688
